packages/components/src/components/RunDropdown/RunDropdown.js

```diff
@@ -42,42 +42,46 @@ class RunDropdown extends Component {
 
     return (
       <>
-        <OverflowMenu
-          title={intl.formatMessage({
-            id: 'dashboard.list.menu.tooltip',
-            defaultMessage: 'Actions'
-          })}
-          data-testid="overflowmenu"
-          flipped
-        >
-          {items.map(item => {
-            const { actionText, action, disable, modalProperties } = item;
-            const disabled = disable && disable(resource);
-            return (
-              <OverflowMenuItem
-                key={actionText}
-                itemText={actionText}
-                disabled={disabled}
-                onClick={() =>
-                  this.handleShow(() => action(resource), modalProperties)
-                }
-                primaryFocus={!disabled}
-              />
-            );
-          })}
-        </OverflowMenu>
-        <Modal
-          open={showDialog}
-          modalHeading={modal.heading}
-          primaryButtonText={modal.primaryButtonText}
-          secondaryButtonText={modal.secondaryButtonText}
-          onRequestClose={this.handleClose}
-          onRequestSubmit={this.handleModalAction}
-          onSecondarySubmit={this.handleClose}
-          danger={modal.danger}
-        >
-          {modal.body && modal.body(resource)}
-        </Modal>
+        {items.length > 0 && (
```

Comment:
we should check this where the RunDropdown is rendered and skip it instead as in https://github.com/tektoncd/dashboard/pull/1084/files#diff-2989be414084b3e10b098bac6149761fR190